src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -3082,7 +3082,7 @@ public function setRawAttributes(array $attributes, $sync = false)
      *
      * @param  string|null  $key
      * @param  mixed  $default
-     * @return array
+     * @return mixed|array
```

Comment:
not correct, the set of arrays are a strict subset of the universal set, "mixed"